schema/alexandria_convention/snapshot/1.1.4/alexandria_convention_schema.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$id": "https://singlecell.broadinstitute.org/single_cell/api/v1/metadata-schema/1.1.4/alexandria_convention_schema.json",
+    "$id": "https://singlecell.broadinstitute.org/single_cell/api/v1/metadata_schemas/1.1.4/json",
```

Comment:
This should read `/api/v1/metadata_schemas/alexandria_convention` - missing the project name.

Reply:
Haha, that's what I get for making manual changes rather than running the script. Fixed now in `ac51f71`